skimage/io/tests/test_pil.py

```diff
@@ -212,6 +212,23 @@ def test_imsave_filelike():
     assert_allclose(out, image)
 
 
+def test_imsave_boolean_input():
+    shape = (2, 2)
+    image = np.zeros(shape)
+    s = BytesIO()
+
+    # save to file-like object
+    with expected_warnings(['is a low contrast image',
+                            'is a boolean image: setting True to 1 and False to 0']):
+        imsave(s, image.astype(bool))
```

Comment:
That's not a low-contrast image at all!

Reply:
Actually it is because there are no 1s in it. I would suggest that this test should set one value to `True` and make sure that that warning is *not* raised.